readme.md

```diff
@@ -72,7 +72,7 @@ The following dependencies need to be installed on your system:
 				* Desktop development with C++
 			* Then in the Installation details tree view, under Desktop for C++, Optional, ensure the following are selected:
 				* MSVC v142 - VS 2019 C++ x64/x86 build tools
-				* Windows 11 SDK (10.0.22000.0)
+				* Windows 11 SDK (10.0.22621.0)
```

Comment:
It might be wise to wait for this version of SDK being available on AppVeyor before asking users to use it locally when building, just to make sure there are no unexpected regressions.

Reply:
What is used by AppVeyor @josephsl ?